brick/__main__.py

```diff
@@ -246,9 +248,8 @@ def list_(ctx, target):
 
 @cli.command()
 @click.argument("target", default=".")
-@click.argument("skip_previous_steps", default=False)
```

Comment:
This doesn't make sense as the flag is happening before the command (e.g. `build`):
`brick [flags] [command] [arguments]`
The argument is therefore never read.